docs/proposals/20220330-topology-mutation-hook.md

```diff
@@ -160,7 +163,7 @@ spec:
     ...
 ```
 
-If the External Patch Extension requires variable definitions, they have to be added to ClusterClass.spec.variables. It is up to the External Patch Extension developer to document them including their OpenAPI schema. As future work we will explore a solution to discover variable definitions from External Patch Extensions automatically.
+If the External Patch Extension requires variable definitions, they have to be added to either to ClusterClass.spec.variables or defined and supplied using a Variable Discovery Hook. It is up to the External Patch Extension developer to document them including their OpenAPI schema.
```

Comment:
Similar here "they have to be added to either to ClusterClass.spec.variables" won't work anymore (as we don't send them to the extension)